src/index.html

```diff
@@ -17,6 +17,50 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <img src="./images/logo.png"
+           class="img"
+           alt="moyologo">
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <div src="./images/Rectangle 2.svg"
+               class="rect"
+               alt="line">>
+            <ul class="apple is-active"
+              href="apple.html">APPLE
+            </ul>
```

Comment:
The `<ul>` element is incorrectly used here. If you intend to create a navigation link, consider using an `<li>` element within a `<ul>` or `<nav>` and wrap the text in an `<a>` tag with the `href` attribute.